install_config/install/deploy_router.adoc

```diff
@@ -350,12 +350,12 @@ named `www.example.test`, see the example shown below:
       -signkey example-test.key -out example-test.crt
 ----
 
-Generate a route configuration file using the above certificate and key.
-Make sure to replace servicename `my-service` with the name of your service.
+Generate a route using the above certificate and key.
 
 ----
-# servicename="my-service"
-# echo "
+$ oc create route edge secured-edge-route --service=my-service --hostname=www.example.test --key=example-test.key --cert=example-test.crt
```

Comment:
Will split the flags into separate lines though I prefer to keep `=` because it makes it explicit that this is not a boolean flag:)